actionpack/lib/action_controller/metal/head.rb

```diff
@@ -50,8 +50,8 @@ def head(status, options = {})
     end
 
     private
-    # :nodoc:
-    def include_content?(status)
+
+    def include_content?(status) # :nodoc:
```

Comment:
Do we really need to mark the method with `nodooc`, even though the method' private? I believe private methods do not show up in documentation.